interfaces/builtin/desktop.go

```diff
@@ -131,6 +131,32 @@ dbus (send)
   member="Get{,All}"
   peer=(label=unconfined),
 
+# Allow accessing the GNOME crypto services prompt APIs as used by
+# applications using libgcr (such as pinentry-gnome3) for secure pin
+# entry to unlock GPG keys etc. See:
+# https://developer.gnome.org/gcr/unstable/GcrPrompt.html
+# https://developer.gnome.org/gcr/unstable/GcrSecretExchange.html
+dbus (send)
+    bus=session
+    path=/org/gnome/keyring/Prompter
+    interface=org.gnome.keyring.internal.Prompter
+    member="{BeginPrompting,PerformPrompt,StopPrompting}"
+    peer=(label=unconfined),
+
+# Whilst this path is not snap specific (ie. any snap or other application
+# could register it on the session bus) this is safe to allow snaps to
+# receive via this since the snap itself has to provide this path name via
+# the BeginPrompting/PerformPrompt methods above to initiate a prompt in
+# the first place so it has complete control to first register this path on
+# the session bus ensure no other application/snap can do so and possibly
+# intercept their response
```

Comment:
Since the testsuite passed, that gives us the opportunity to make this slightly less wordy. :) How about:
```
# While the DBus path is not snap-specific, by the time an application
# registers the prompt path via DBus, Gcr will check that it isn't
# already in use and send the client an error. See:
# https://github.com/snapcore/snapd/pull/7673#issuecomment-592229711
```

Reply:
Thanks - yep this is a bit less wordy :)